media/js/firefox/android.js

```diff
@@ -23,6 +23,11 @@
         timeout: 2000
     };
 
+    // send to device vars
+    var hasWidget = $('#intro').data('has-widget') !== 'False';
```

Comment:
Nit: even though jQuery is relax on data-attributes, I believe the standards way is to use camelCase: `$('#intro').data('hasWidget')`